companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greeting2s",
     "relevant-countries": [
-        "all"
+        "all",
+        "dong"
     ],
     "categories": [
         "entertainment"
     ],
     "name": "123Greetings.com Inc.",
-    "address": "255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
+    "address": "123Greetings.com Inc.\n255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
     "fax": "+1 212 202 4738",
     "email": "privacy@123greetings-inc.com",
-    "web": "https://www.123greetings.com",
+    "web": "%https://www.123greetings.com",
     "sources": [
         "https://www.123greetings.com/privacy_policy.html",
         "https://www.123greetings.com/terms_of_use.html",
         "https://github.com/datenanfragen/data/pull/1025"
     ],
-    "needs-id-document": false,
+    "needs-id-document": "ja",
     "suggested-transport-medium": "email",
-    "quality": "verified"
+    "quality": "tested"
```

Comment:
🚫 **[companylint]** <[tested-requirements](https://github.com/datenanfragen/data/issues/811)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record has `quality` of `tested` but doesn't specify `required-elements`.

Preferably set `required-elements`, otherwise demote to `verified`.
```suggestion
    "quality": "verified"
```